test/e2e/user_group_limit/user_group_limit_test.go

```diff
@@ -641,6 +641,40 @@ var _ = ginkgo.Describe("UserGroupLimit", func() {
 		checkUsage(groupTestType, url.QueryEscape(validGroup), sandboxQueue1, []*v1.Pod{usergroup1Sandbox1Pod1})
 	})
 
+	ginkgo.It("Verify_Queue_Name_With_Special_Characters", func() {
+		ginkgo.By("Create a queue with a name that includes all allowed special characters")
+		queueName := "queue-#_@/:"
```

Comment:
You can include numbers as well. or May be we can reuse the variables from configmap_test.go (Line No. 115) by keeping it in common place.

Reply:
Updated with the below Variable 
`root_test22-a_b_#_c_#_d__e@dom:ain`